source/agora/test/Base.d

```diff
@@ -1101,9 +1101,11 @@ public class TestAPIManager
                     (file, line, height, idx,
                         this.clients[idx].getBlockHeight())));
 
```

Comment:
there might be a little bit of race condition here:

1. at line 1097 all clients has a block height of 10 including client 1, however client 1 is too busy to respond to the testing framework
3. all clients almost ready for accepting block with height 11
4. client 1 is finally able to report block height 10, so line 1097 returns
5. all clients accepted block with height 11
6. the assert will trigger at 1108

this has probably low chance to manifest, but we have big timing delays in test and small block creation(1sec/block I think)